Library/ViewModels/RewardPledgeViewModel.swift

```diff
@@ -361,9 +376,15 @@ RewardPledgeViewModelOutputs {
       }
       .skipNil()
 
-    self.fulfillmentAndShippingFooterStackViewHidden = reward
+    self.estimatedFulfillmentStackViewHidden = reward
+      .map { $0.estimatedDeliveryOn == nil }
+
+    self.shippingStackViewHidden = reward
       .map { !$0.shipping.enabled }
 
+    self.fulfillmentAndShippingFooterStackViewHidden = reward
+      .map { $0.estimatedDeliveryOn == nil && !$0.shipping.enabled }
```

Comment:
previously, we were hiding the entire shipping/fulfillment stack view if shipping was disabled, but now we want to always show delivery estimate if it's valid.